frontend/src/container/CreateAlertRule/defaults.ts

```diff
@@ -130,7 +130,7 @@ export const exceptionAlertDefaults: AlertDef = {
 					disabled: false,
 				},
 			},
-			queryType: EQueryType.QUERY_BUILDER,
+			queryType: EQueryType.CLICKHOUSE,
```

Comment:
is this expected to change default to Clickhouse, other places have queryBuilder only i guess

Reply:
yes this is expected

Reply:
for exceptions the default is clickhouse